src/styles/blocks/header.scss

```diff
@@ -38,5 +36,10 @@
     text-transform: uppercase;
     font-size: 12px;
     font-weight: 500;
+    transition-duration: $animation-duration;
```

Comment:
add transition rule and specify the properties you change on hover

Reply:
As I sure understand, it must be changed menu-link text-color, when it is hovered.
![image](https://user-images.githubusercontent.com/126285904/230768822-6cf79327-f48f-4bb8-a150-c3e0100c7d20.png)
I used "color" in ":hover" and "transition" for implement transition-duration. Is it wrong?  It really works, I sweare I ckecked it :)

Reply:
yeah it works, but it's better for browser to specify exactly which properties you change on hover

Reply:
But if I set specify, when hover has only one property - it doesn't work

Reply:
sorry, forgot to delete "duration" from operator